dask/tests/test_order.py

```diff
@@ -409,3 +409,95 @@ def test_string_ordering_dependents():
                  ('a', 1): 1,
                  ('a', 2): 2,
                  ('a', 3): 3}
+
+
+def test_prefer_short_narrow():
```

Comment:
I recommend using the abcde fixture here to use that for key names.  It changes order to ensure that we're not entirely dependent on the fact that, `'a' < 'b'` for example.